vmdb/app/models/ems_kubernetes.rb

```diff
@@ -0,0 +1,21 @@
+class EmsKubernetes < EmsContainer
+  has_many :container_nodes,                      :foreign_key => :ems_id, :dependent => :destroy
+  has_many :container_groups,                       :foreign_key => :ems_id, :dependent => :destroy
+  has_many :container_services,                   :foreign_key => :ems_id, :dependent => :destroy
```

Comment:
Can you clean up the whitespace here?  I think you can align the `:foreign_key` part still, just one space after the longer entry.